src/audiomixerboard.cpp

```diff
@@ -1305,6 +1307,49 @@ void CAudioMixerBoard::SetAllFaderLevelsToNewClientLevel()
     }
 }
 
+void CAudioMixerBoard::AutoAdjustAllFaderLevels()
+{
+    QMutexLocker locker ( &Mutex );
+
+    for ( int i = 0; i < MAX_NUM_CHANNELS; ++i )
+    {
+        // only apply to visible faders (and not to my own channel fader)
+        if ( vecpChanFader[i]->IsVisible() && ( true /* i != iMyChannelID */ ) )
```

Comment:
I'm guessing this is temporary `true /* i != iMyChannelID */`?

Reply:
Yes, I will switch to the originally intended behavior such that it is the same as for `Set All Faders to New Client Level`.

Reply:
Yes, I will switch to the originally intended behavior such that it is the same as for `Set All Faders to New Client Level`.